reconcile/utils/gql.py

```diff
@@ -108,18 +114,16 @@ def __init__(self, url, token=None, int_name=None, validate_schemas=False):
                 raise GqlApiIntegrationNotFound(int_name)
 
     @retry(exceptions=GqlApiError, max_attempts=5, hook=capture_and_forget)
-    def query(self, query, variables=None, skip_validation=False):
+    def query(self, query, variables=None, skip_validation=False) -> Dict[str, Dict]:
```

Comment:
No need for the abstract `Dict` here. Use `dict` as the return type.

Reply:
addressed